src/build-request-url-params.ts

```diff
@@ -0,0 +1,20 @@
+// Imports
+import { Options } from './types/options';
+
+// Helper function that takes in parameters and builds a URL to make a request with
+function buildRequestURLParams(parameters: Options & { url: string }): string {
+	// Return request URL
```

Comment:
We should probably either...
1. Remove this comment, since it doesn't add much value on its own.
2. Move this comment down to the `return` statement, and add a more descriptive comment that describes the `const params =` statement.